platform/mac/mcreator.sh

```diff
@@ -1,14 +1,16 @@
-#!/bin/bash
-#
+#!/bin/zsh
+
 # ---------------------------------------------------------------------
-# MCreator Launcher file for MacOS
+# MCreator Launcher file for macOS
+# zsh is now used due to bash being missing on newer macOS distributions
 # ---------------------------------------------------------------------
-#
 
-# WARNING: WHEN CHANGING THIS FILE, "mcreator" binary needs to be recompiled using "shc -v -r -f mcreator.sh"
+# WARNING: WHEN CHANGING THIS FILE, MCREATOR BINARIES NEED TO BE RECOMPILED WITH SHC (UNDER RELAXED MODE)
+# To compile the intel binary on an aarch64 machine, re-brew shc under rosetta
+# Use the prefix /opt/homebrew/bin/shc for aarch64 and /usr/local/bin/shc for x86_64
```

Comment:
Please provide whole command lines for convenience here, eg.

`/opt/homebrew/bin/shc -v -r -f mcreator.sh`

or whatever you used, so this is 100% reproducible by other users